handbook/data-stores/mongodb.md

```diff
@@ -0,0 +1,133 @@
+
+### Increasing the index cache size for *single node* deployments
+
+Increase the index cache size to process more data and use the Test Insights or Asset Management modules in a single-node deployment. This avoids high CPU usage that can occur with the default MongoDB memory constraint in SystemLink. NI recommends increasing the available memory for the index to 8GB. Refer to [Sizing a SystemLink Server](https://www.ni.com/documentation/en/systemlink/latest/setup/sizing-a-systemlink-server/) for system requirements to run SystemLink.
+  
+1. Log into the desktop of the SystemLink app server with a user with administrator privileges.
+1. Open **NI SystemLink Server Configuration**.
+1. Navigate to **NoSqlDatabase**.
+1. Under **The NoSqlServer is launched and managed by SystemLink**, adjust the value in **The cache size of the database in Gigabytes (GB)**.
+1. Click **Apply**.
+
+<figure>
+  <img src="../../img/mdb-cache.png" width="500" />
+  <figcaption>SystemLink's NoSqlDatabase configuration set to 8BG</figcaption>
+</figure>
+
+## Multi Node Deployments
+
+SystemLink supports three types of multi-node configuration. You can configure SystemLink to connect to a standalone instance, replica sets, or MongoDB Atlas.
+
+NI recommends connecting SystemLink to replica set of three or more MongoDB members or MongoDB Atlas for all production deployments.
+
+- NI does not support configurations where multiple SystemLink servers use the same MongoDB instance.
+- NI does not support configurations where another production application uses the same MongoDB instance as SystemLink.
+- While SystemLink can connect to and use a sharded MongoDB cluster (`mongos`), it will not take advantage of horizontal scaling capabilities enabled by sharded clusters.
+- When using MongoDB Atlas or [MongoDB Enterprise Advanced](https://www.mongodb.com/products/mongodb-enterprise-advanced), you have the option to encrypt the data stored within the database.
+
+!!!important "Connection String Formats"
+    Some connection string formats are unsupported or could cause issues in some environments.
+
+    - Connection strings that contain the query parameter `tls=` are unsupported. Use the `ssl=` query string to achieve the same degree of security.
+    - In environments where the 4.2.1.1 root DNS server cannot be reached, the DNS seed list URI format (`mongodb+srv://`) will fail due to a bug ([Unable to connect to Atlas due to DNS connectivity issues #358](https://github.com/kobil-systems/mongodb/issues/358)) in a 3rd party MongoDB driver used by SystemLink. In this case use the [MongoDB standard connection string format](https://docs.mongodb.com/manual/reference/connection-string/#std-label-connections-standard-connection-string-format) to connect your replica set. This affects both self-hosted replica sets and MongoDB Atlas.
```

Comment:
```suggestion
    - In environments where the 4.2.1.1 root DNS server cannot be reached, the DNS seed list URI format (`mongodb+srv://`) will fail due to a bug ([Unable to connect to Atlas due to DNS connectivity issues #358](https://github.com/kobil-systems/mongodb/issues/358)) in a third-party MongoDB driver SystemLink uses. In this case, use the [MongoDB standard connection string format](https://docs.mongodb.com/manual/reference/connection-string/#std-label-connections-standard-connection-string-format) to connect your replica set. This affects both self-hosted replica sets and MongoDB Atlas.
```

Reply:
I fixed the IP, and dropped "SystemLink uses" since that should be obvious.